pkg/containerutil/cp_linux.go

```diff
@@ -54,13 +55,27 @@ func CopyFiles(ctx context.Context, client *containerd.Client, container contain
 		if !errdefs.IsNotFound(err) || rootlessutil.IsRootless() {
 			return err
```

Comment:
possible to add a message to mention that copying to a container without root is not yet supported ?